automation/taskcluster/androidTest/ui-test.sh

```diff
@@ -84,8 +84,15 @@ samples=${component//samples-}
 if [[ "${component}" != samples-* ]]
 then
     # Case 1: tests for any component (but NOT samples, NOT real UI tests)
-    APK_APP="./samples/${component}/build/outputs/apk/geckoNightly/debug/samples-${component}-geckoNightly-debug.apk"
-    APK_TEST="./components/${component}/engine-gecko-nightly/build/outputs/apk/androidTest/debug/browser-engine-gecko-nightly-debug-androidTest.apk"
+    APK_APP="./samples/browser/build/outputs/apk/geckoNightly/debug/samples-browser-geckoNightly-debug.apk"
+    if [[ "${component}" == *"-"* ]]
```

Comment:
If I don't find component like` feature-*`, `concept-*`, `browser-*` ... etc I fallback to the previous implementation